packages/app/project.json

```diff
@@ -36,7 +36,7 @@
       ]
     },
     "serve": {
-      "executor": "@nx/node:node",
```

Comment:
This is the issue I was talking about a few days back. The live serve was broken.